package-managers/nuget.md

```diff
@@ -86,21 +94,67 @@ After making sure our feed is a public one we can consume it in Rider. To work w
 
 ### NuGet CLI
 
+NuGet package restore relies on the [NuGet.exe](https://nuget.org/nuget.exe) commandline tool by using the [install](https://docs.microsoft.com/nuget/tools/nuget-exe-cli-reference#Install_Command) command. The commandline will either prompt us for credentials or will look for credentials in the `NuGet.config` file in `%AppData%\NuGet\nuget.config` (if we use the Non-Interactive option).
+
 #### Private feed
 
 We will need the URL for the feed we want to connect to. Learn more about [determining the feed URL](#determining-the-feed-url).
 
+To register our credentials in the `NuGet.config` file we use the following commands:

```

Comment:
"we use the following commands"--> we can/will/... use the following commands
(misses a verb)